docs/qa/CometBFT-QA-38.md

```diff
@@ -359,8 +359,216 @@ number of transactions processed per minute as compared to the baseline.
+| **4k**       | ![cpu-avg](img38/voteExtensions/04k_avg_cpu.png)      |
+| **8k**       | ![cpu-avg](img38/voteExtensions/08k_avg_cpu.png)      |
+| **16k**      | ![cpu-avg](img38/voteExtensions/16k_avg_cpu.png)      |
+| **32k**      | ![cpu-avg](img38/voteExtensions/32k_avg_cpu.png)      |
+
+### Resident Memory
+
+The same conclusion reached for CPU usage may be drawn for the memory.
+That is, that a backlog of work is formed during the tests and catching up (freeing of memory) happens after the test is done.
+
+> :warning:
+> A more worrying trend is that the bottom of the memory usage seems to increase in between runs.
+> This will be investigated in a future issue.
+
+
+| Experiment   | Resident Set Size                                        |
+| ------------ | -------------------------------------------------------- |
+| **baseline** | ![rss-avg](img38/voteExtensions/baseline_avg_memory.png) |
+| **8k**       | ![rss-avg](img38/voteExtensions/02k_avg_memory.png)      |
+| **8k**       | ![rss-avg](img38/voteExtensions/04k_avg_memory.png)      |
+| **8k**       | ![rss-avg](img38/voteExtensions/08k_avg_memory.png)      |
+| **16k**      | ![rss-avg](img38/voteExtensions/16k_avg_memory.png)      |
+| **32k**      | ![rss-avg](img38/voteExtensions/32k_avg_memory.png)      |
+
+### Mempool size
+
+This metric shows how many transactions are outstanding in the nodes' mempools.
+Observe that in all runs, the average number of transactions in the mempool quickly drops to near zero between runs.
+
+
+| Experiment   | Resident Set Size                                                  |
+| ------------ | ------------------------------------------------------------------ |
+| **baseline** | ![mempool-avg](img38/voteExtensions/baseline_avg_mempool_size.png) |
+| **8k**       | ![mempool-avg](img38/voteExtensions/02k_avg_mempool_size.png)      |
+| **8k**       | ![mempool-avg](img38/voteExtensions/04k_avg_mempool_size.png)      |
+| **8k**       | ![mempool-avg](img38/voteExtensions/08k_avg_mempool_size.png)      |
+| **16k**      | ![mempool-avg](img38/voteExtensions/16k_avg_mempool_size.png)      |
+| **32k**      | ![mempool-avg](img38/voteExtensions/32k_avg_mempool_size.png)      |
+
+
+
+#### Observations/Questions/TODOs
+
+* what is the effect on block sizes
+    * how is the block size limited? do the extensions fit? the proposal itself is small.
+    * block sizes depending on the extension sizes
+* what is the effect on the network usage
+* txs per block depending on the extension sizes
+    * remember that there is one special transaction
+* we need to evaluate the effect of transaction sizes as well.
+* the mempool never got completely empty on some nodes
+* 5 minute interval to clean up the mempool. heights continue to happen, with vote extensions.
+* if the mempool size drops quickly, why is the memory slow to empty? Slow GC?
+* run with longer timeouts
+* run baseline for 20 minutes straight.
```

Comment:
Could we move this text to a (tracking) issue?

Reply:
Done 903.